gravity.sh

```diff
@@ -872,12 +889,44 @@ gravity_Cleanup() {
   fi
 }
 
+switch_database() {
+  if [[ -f "${gravityOLDfile}" ]]; then
+    gravityOLDversion=$(pihole-FTL "${gravityOLDfile}" "SELECT value FROM info WHERE property IS 'version';")
+    echo -e "  ${TICK}  ${gravityOLDfile} exists."
+  else
+    echo -e "  ${CROSS}  Cannot switch database, ${gravityOLDfile} doesn't exist."
+    exit 1
+  fi
+  if [[ -f "${gravityDBfile}" ]]; then
+    echo -e "  ${TICK}  ${gravityDBfile} exists."
+    if [[ ${gravityOLDversion} -eq $(pihole-FTL "${gravityDBfile}" "SELECT value FROM info WHERE property IS 'version';") ]]; then
+      echo -e "  ${INFO}  Switching database."
+      echo "         current database, updated $(date -d @"$(pihole-FTL "${gravityDBfile}" "SELECT value FROM info WHERE property IS 'updated';")")"
+      oldDBdate=$(pihole-FTL "${gravityOLDfile}" "SELECT value FROM info WHERE property IS 'updated';")
+      echo "         old database, updated $(date -d @"${oldDBdate}")"
+      mv "${gravityOLDfile}" "$(dirname -- "${gravityOLDfile}")/gravity_mv.db"
```

Comment:
This might be a good place to add a check for disk space again?

Reply:
I'm open to all suggestions, but why should I check the diskspace here? No additional diskspace is required...
What happens is:
- rename gravity_old.db -> gravity_mv.db
- rename gravity.db -> gravity_old.db
- rename gravity_mv.db -> gravity.db
End result is a name switch, nothing more.
Are you suggesting to simply remove gravity_old.db, if there isn't enough diskspace, and thus, not perform a switch?

Reply:
You're right. I mixed it up.